google-assistant-grpc/CHANGELOG.rst

```diff
@@ -0,0 +1,10 @@
+Changelog
+=========
+
+0.0.2
+-----
+- Bump ``grpcio`` dependency.
```

Comment:
Can you explain what was the benefit and implications of doing this? (in one line)